examples/web_agent/worker.py

```diff
@@ -0,0 +1,203 @@
+import os
+import json
+import asyncio
+from airtop import Airtop, SessionConfigV1, PageQueryConfig
+from dotenv import load_dotenv
+
+from prompt_templates import comparison_prompt
+from utils import (
+    find_workers,
+    create_worker,
+    retrieve_previous_result,
+    insert_result,
+    insert_comparison,
+)
+from connect_database import (
+    sql_statements,
+    insert_tables,
+    dataframe_from_query_dev,
+)
+
+load_dotenv()
+
+AIRTOP_API_KEY = os.getenv("AIRTOP_API_KEY")
+LOGIN_URL = "https://www.glassdoor.com/member/profile"
+IS_LOGGED_IN_PROMPT = "This browser is open to a page that either display's a user's Glassdoor profile or prompts the user to login.  Please give me a JSON response matching the schema below."
+IS_LOGGED_IN_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "isLoggedIn": {
+            "type": "boolean",
+            "description": "Use this field to indicate whether the user is logged in.",
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem.",
+        },
+    },
+}
+TARGET_URL = "https://www.glassdoor.com/Job/san-francisco-ca-software-engineer-jobs-SRCH_IL.0,16_IC1147401_KO17,34.htm"
+EXTRACT_DATA_PROMPT = "This browser is open to a page that lists available job roles for software engineers in San Francisco. Please provide 10 job roles that appear to be posted by the AI-related companies."
+EXTRACT_DATA_OUTPUT_SCHEMA = {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "companies": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "companyName": {"type": "string"},
+                    "jobTitle": {"type": "string"},
+                    "location": {"type": "string"},
+                    "salary": {
+                        "type": "object",
+                        "properties": {
+                            "min": {"type": "number", "minimum": 0},
+                            "max": {"type": "number", "minimum": 0},
+                        },
+                        "required": ["min", "max"],
+                    },
+                },
+                "required": ["companyName", "jobTitle", "location", "salary"],
+            },
+        },
+        "error": {
+            "type": "string",
+            "description": "If you cannot fulfill the request, use this field to report the problem.",
+        },
+    },
+}
+
+
+async def run():
```

Comment:
So you can remove the asyncio import, change to:
```suggestion
def run():
```

Reply:
Ok, removed the asyncio and this async def